mlflow/sklearn.py

```diff
@@ -30,13 +28,16 @@
 
 FLAVOR_NAME = "sklearn"
 
-DEFAULT_CONDA_ENV = _mlflow_conda_env(
-    additional_conda_deps=[
-        "scikit-learn={}".format(sklearn.__version__),
-    ],
-    additional_pip_deps=None,
-    additional_conda_channels=None,
-)
+
+def get_default_conda_env():
```

Comment:
I think we should add method docs here since this is a public method

Reply:
I've filed https://github.com/mlflow/mlflow/pull/1238/ which makes this change and adds docs for all relevant flavors, including `sklearn`.